### PR TITLE
[NO_JIRA] Few actions were made:

### PR DESCRIPTION
**Pull request by rpkyrych**
* removed excess imports
* removed excess annotations
* removed excess tests
* removed useless methods
* removed useless messages
* reduced some warnings
* refactored some duplications

### CODE REVIEW

**Review by Wolshebnik — APPROVED** [review]

**Review by MaryGoroshko — APPROVED** [review]

File: src/main/java/space/obminyashka/items_exchange/dto/AdvertisementModificationDto.java [hunk 1]
```diff
@@ -25,7 +25,6 @@
 
     @Positive(groups = Exist.class, message = "{invalid.exist.id}")
     @Zero(groups = New.class, message = "{new.advertisement.id.not-zero}")
-    @NotNull(message = "{invalid.not-null}")
```
Comment (Jack11M) [inline src/main/java/space/obminyashka/items_exchange/dto/AdvertisementModificationDto.java:28]: We use this DTO also when create adv.
Its like a validation. 
How will we notify front about incorrect value
 without it?

File: src/main/java/space/obminyashka/items_exchange/dto/AdvertisementModificationDto.java [hunk 2]
```diff
@@ -48,7 +47,6 @@
     @NotNull(message = "{invalid.not-null}")
     private AgeRange age;
 
-    @NotNull(message = "{invalid.not-null}")
```
Comment (Jack11M) [inline src/main/java/space/obminyashka/items_exchange/dto/AdvertisementModificationDto.java:51]: The same as above

File: src/main/java/space/obminyashka/items_exchange/dto/AdvertisementModificationDto.java [hunk 3]
```diff
@@ -61,9 +59,6 @@
     @Size(min = 1, max = 50, message = "{invalid.size}")
     private String size;
 
-    @NotNull(message = "{invalid.not-null}")
```
Comment (Jack11M) [inline src/main/java/space/obminyashka/items_exchange/dto/AdvertisementModificationDto.java:64]: We couldn't create a new adv if this id is null, could we?

File: src/main/java/space/obminyashka/items_exchange/dto/AdvertisementModificationDto.java [hunk 4]
```diff
@@ -61,9 +59,6 @@
...
     private long subcategoryId;
-
-    @NotNull(message = "{invalid.not-null}")
```
Comment (Jack11M) [inline src/main/java/space/obminyashka/items_exchange/dto/AdvertisementModificationDto.java:67]: The same as above

File: src/main/java/space/obminyashka/items_exchange/mapper/UtilMapper.java [hunk 5]
```diff
@@ -31,17 +32,18 @@ public void setModelMapper(ModelMapper modelMapper) {
...
     public static <T, K> K convertTo(T src, Class<K> kClass) {
+        mapper.getConfiguration().setMatchingStrategy(MatchingStrategies.LOOSE);
         return mapper.map(src, kClass);
     }
 
     public static <T, K> List<K> convertToDto(Collection<T> tCollection, Class<K> kClass) {
         return tCollection.stream()
                 .map(t -> mapper.map(t, kClass))
-                .collect(Collectors.toList());
+                .toList();
```
Comment (Jack11M) [inline src/main/java/space/obminyashka/items_exchange/mapper/UtilMapper.java:42]: Super!

**Pull request merged**